index.js

```diff
@@ -0,0 +1,13 @@
+let divide = (a, b) => {
+  let reuslt = a / b;
+  return reuslt;
+}
+let square = n => n * 2;
+  alert( square(2 ));
+
+  const add = (a,b) => {
+    console.log(`Adding ${a}`);
+    console.log(`Adding ${b}`);
```

Comment:
Don't forget to remove `console.log`s, `alert`s, and other debugging tools before you submit!